edb/schema/functions.py

```diff
@@ -646,10 +644,12 @@ def param_is_inherited(
                 p for p in new_params
                 if not param_is_inherited(new_schema, new, p)
             ]
+            add = delta.add_prerequisite
```

Comment:
nit: I'd call right here -- the code would be simpler (albeit a bit more repetitive).